exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Table.kt

```diff
@@ -792,9 +792,26 @@ open class Table(name: String = ""): ColumnSet(), DdlAware {
     }
 }
 
-data class Seq(private val name: String) {
```

Comment:
Please deprecate that class with Error level and a link to a new Sequence. 
I don't think that someone really uses it, but it's just a good manners.

Reply:
Ok 👍